docs/getting-started/cli/subsembly-spec.md

```diff
@@ -16,27 +16,140 @@ subsembly spec [to] [src] [raw] [wasm]
 
 #### Parameters
 
-* **`--to`** - \[optional\] initialisation path for the new chain specification file
-* **`--src`** - \[optional\] source specification file to convert to raw, defaults to current directory
-* **`--raw`** - \[optional\] path to save converted raw chain specification file. Defaults to **`{srcPath}/raw-chain-spec.json`**
-* **`--wasm`** - \[optional\] path to the compiled hex encoded **`wasm`** binary. Defaults to **`build/subsembly-wasm`**
+* **`--to`** - optional parameter, determines initialisation path for the new chain specification file, defaults to current directory
+* **`--src`** - optional parameter, determines source specification file to convert to raw, defaults to current directory
+* **`--raw`** - optional parameter, determines path to save converted raw chain specification file. Defaults to **`{path-of-src}/raw-chain-spec.json`**
+* **`--wasm`** - optional parameter, determines path to the compiled hex encoded **`wasm`** binary. Defaults to **`build/subsembly-wasm`**
 
 ### Examples
 
-```text
+{% tabs %}
+{% tab title="1st" %}
```

Comment:
Instead of naming them `1st`, `2nd` etc. we could name them `Option 1` `Option 2`